charts/trino/values.yaml

```diff
@@ -727,6 +727,54 @@ kafka:
   #    }
   # ```
 
+jmx:
+  # https://trino.io/docs/current/admin/jmx.html
+  enabled: false
+  registryPort: 9080
+  serverPort: 9081
+
+jmxExporter:
+  enabled: false
+  image: bitnami/jmx-exporter:latest
+  pullPolicy: Always
+  port: 5556
+  configProperties: []
+  # configProperties -- JMX Config Properties is mounted to /etc/jmx-exporter/jmx-exporter-config.yaml
+  # @raw
+  # Example:
+  # ```yaml
+  #  configProperties: |-
+  #     startDelaySeconds: 0
+  #     ssl: false
+  #     lowercaseOutputName: false
+  #     lowercaseOutputLabelNames: false
+  #     includeObjectNames: ["java.lang:type=Threading"]
+  #     autoExcludeObjectNameAttributes: true
+  #     excludeObjectNameAttributes:
+  #       "java.lang:type=OperatingSystem":
+  #         - "ObjectName"
+  #       "java.lang:type=Runtime":
+  #         - "ClassPath"
+  #         - "SystemProperties"
+  #     rules:
+  #       - pattern: 'java\.lang<type=Threading><(.*)>ThreadCount: (.*)'
+  #         name: java_lang_Threading_ThreadCount
+  #         value: '$2'
+  #         help: 'ThreadCount (java.lang<type=Threading><>ThreadCount)'
+  #         type: UNTYPED
+
+podMonitor:
+  # - podMonitor is a resource from prometheus-operator:
+  #   https://github.com/prometheus-operator/prometheus-operator
```

Comment:
```suggestion
  # podMonitor.enabled -- set to true to create resources for the [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator).
```

Reply:
Sorry I missed it initially, but please start sentences with a capital letter, so it'll be `Set to true...`